### DISCUSSION TITLE
Apk cleanup?

### DISCUSSION THREAD

**Discussion by jcperil** [ImranR98/Obtainium#548] [General]
What happens to the apks downloaded to install? Is there a clean up duration after which the older apks are deleted?

**Comment by ImranR98** [comment]
APKs are automatically deleted after install, but only if the install is successful. If not, they are kept so you can try again without redownloading.

You're right though, there should be an expiration date. The user might change their minds about installing or even remove the app from Obtainium completely, and as it stands, the APK just stays there until the user manually clears the cache.

**Comment by momobobe** [comment]
How about making it an option for the user to decide? Clean right after installation, or in one day, one week or even never, just let the user to decide.
